backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -418,6 +418,88 @@ async def upload_projects_excel_sheet(
             status_code=500,
             detail=f"Failed to process project data: {str(e)}"
         )
+    finally:
+        if input_temp_file and os.path.exists(input_temp_file.name):
+            os.unlink(input_temp_file.name)
+
+@router.post('/facilitySelection',
+             summary='Upload and process facility selection Excel file',
+             response_description='Returns processed Excel file with validations results')
+async def upload_facility_selection_excel_sheet(
+        facility_selection_file: UploadFile = File(description="Excel file containing facility selection data"),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)